.golangci.example.yml

```diff
@@ -103,6 +103,24 @@ linters-settings:
     # should ignore tests (default false)
     skip-tests: false
 
+  errchkjson:
+    # with check-error-free-encoding set to true, errchkjson does warn about errors
+    # from json encoding functions that are safe to be ignored,
+    # because they are not possible to happen (default false)
+    #
+    # if check-error-free-encoding is set to true and errcheck linter is enabled,
+    # it is recommended to add the following exceptions to prevent from false positives:
+    #
+    #     linters-settings:
+    #       errcheck:
+    #         exclude-functions:
+    #           - encoding/json.Marshal
+    #           - encoding/json.MarshalIndent
+    #           - (*encoding/json.Encoder).Encode
+    check-error-free-encoding: false
+    # if report-no-exported is true, encoding a struct without exported fields is reported as issue (default false)
+    report-no-exported: false
+
```

Comment:
can you move this section to the right position following the alphabetical order?
And add a description of your linter.